source/adapters/native_cpu/enqueue.cpp

```diff
@@ -63,6 +63,23 @@ static native_cpu::state getResizedState(const native_cpu::NDRDescT &ndr,
 }
 #endif
 
+class LaunchInfoLocalArgs {
+  native_cpu::state state;
+  const size_t numParallelThreads;
+
+public:
+  LaunchInfoLocalArgs(const native_cpu::state &state_, unsigned g0_,
```

Comment:
No need for the `_` suffixes, there is no conflict between the parameter and field names, I think we don't do that elsewhere either? For instance, in `kernel.hpp`:
```c++
  ur_kernel_handle_t_(ur_program_handle_t hProgram, const char *name,
                      nativecpu_task_t subhandler)
      : hProgram(hProgram), _name{name}, _subhandler{std::move(subhandler)} {}
```